sns/src/main/scala/akka/stream/alpakka/sns/SnsPublishSettings.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.sns
+
+class SnsPublishSettings(val concurrency: Int) {
```

Comment:
Make the constructor private and the class final.

Reply:
fixed